### ISSUE TITLE
"Edit Document" corrupts document

### ISSUE THREAD

**Issue by fgarcia** [paralect/robomongo#870]
Editing a document with an array of long integers (int32 and int64) like deleting the first rows of this:

    "follower_ids" : [ 
        NumberLong(3243178833), 
        1.96693e+09, 
        2.50076e+07, 
        NumberLong(2433114481), 
        NumberLong(2213529570), 

after clicking save, the new document will change some values of the array from int64 into Float

UPDATE:

After using some other client, I have noticed that  the numbers are rendered in a different way:

    "follower_ids": [
      NumberLong(3243178833),
      1966934329,
      25007589,
      NumberLong(2433114481),
      NumberLong(2213529570),

**Comment by zygis** [comment]
Yes, looks like we must forget robomongo for now... Our team (and our clients) already suffering from corrupted data after using robomongo. Swithed to 3T mongoChef. Robomongo **WAS** nice peace of software, but today is outdated.

**Comment by borigas** [comment]
I think this is a duplicate of #831 and sounds like it has already been fixed. Just waiting on the next version.

**Comment by zygis** [comment]
0.8.4 / November 27, 2013
0.8.5 / March 10, 2015
0.8.6 / Jun 20, 2016 ?? :)

**Comment by schetnikovich** [comment]
@fgarcia @zygis This issue was fixed in just released [Robomongo 0.9.0 RC3](http://blog.robomongo.org/robomongo-0-9-0-rc3-released). Let us know how it works for you.

**Comment by zygis** [comment]
Sorry but I already not using robomongo. This product horribly outdated.
